src/tests/Microsoft.PowerFx.Connectors.Tests/OpenApiParserTests.cs

```diff
@@ -39,10 +41,50 @@ public void ACSL_GetFunctionNames()
             Assert.Equal("Conversations (CLU) (2022-05-01)", function.Summary);
             Assert.Equal("/apim/cognitiveservicestextanalytics/{connectionId}/language/:analyze-conversations", function.OperationPath);
             Assert.Equal(HttpMethod.Post, function.HttpMethod);
-        }
-
+        }
+
+        [Fact(Skip = "Need files from AAPT-connector and PowerPlatformConnectors projects")]

```

Comment:
This seems like a test that shouldn't live here? Or those files should be added to this project. Referring to absolute paths in a test seems like a bad idea, generally we should be using relative paths that are populated by the build.

Reply:
This is mostly 
- dirty, I know that ;)
- only for me, to tests all connectors
I could use env variables too, define a root localtion string, etc. but I don't see the value.

The code is only there to keep it and easily run it on my machine.
I could also add #ifdef CONNECTOR_TESTING if we really want to add some extra protection

Reply:
I'd be fine keeping as a test and moving this to a separate file, and adding a comment to make the intent clear.